.env.sample

```
@@ -0,0 +1,11 @@
+PORT=portnumber
+
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
+
+TYPEORM_CONNECTION=mysql
+TYPEORM_HOST=localhost
+TYPEORM_USERNAME=username
+TYPEORM_PASSWORD=password
+TYPEORM_DATABASE=database_name
+TYPEORM_PORT=portnumber
+TYPEORM_LOGGING=TRUE
```

직접 true/false 값을 기입하기 보단 boolean으로 명시하는 것이 좋을 것 같습니다.
```suggestion
TYPEORM_LOGGING=boolean
```